ci-operator/config/netobserv/network-observability-operator/netobserv-network-observability-operator-main.yaml

```diff
@@ -0,0 +1,57 @@
+base_images:
+  ubi:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.20
+images:
+- dockerfile_path: Dockerfile
+  from: ubi
+  to: network-observability-operator
+operator:
+  bundles:
+  - as: noo-bundle
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/netobserv/network-observability-operator:1.0.4
+    with: network-observability-operator
+promotion:
+  name: "4.14"
```

Comment:
why 4.14?

Reply:
changed to 4.15

Reply:
master/main represents 4.16 currently i believe.

Reply:
see: https://github.com/openshift/release/blob/3d560d554670c6f6eb60ab9aa593a93e7af6f908/ci-operator/config/openshift/cluster-ingress-operator/openshift-cluster-ingress-operator-master.yaml#L29-L32

Reply:
ah okay got it. Will update that